README.md

```diff
@@ -3,6 +3,26 @@ eLife Profiles
 
 [![Build Status](https://ci--alfred.elifesciences.org/buildStatus/icon?job=test-profiles)](https://ci--alfred.elifesciences.org/job/test-profiles/) [![Coverage Status](https://coveralls.io/repos/github/elifesciences/profiles/badge.svg?branch=develop)](https://coveralls.io/github/elifesciences/profiles?branch=develop)
 
+Profiles is a service used by eLife Sciences Publications, Ltd. to create and 
+store eLife user profiles. Currently, eLife uses [ORCID](https://orcid.org/) for 
+authentication.
+
+When a user account is created or updated on ORCID, the new account information is sent to 
+the Profiles service via a webhook and a new eLife profile is created, ready to be
+used by other services.
+
+End points
+----------
+- `/ping` returns the message `pong`
+- `/profiles` returns a list of profiles *(this is paginated by default, 
+sending 20 results per request. Only basic info sent. 
+See [Here](https://api.elifesciences.org) for more information)*
```

Comment:
I would prefer just linking to the API docs instead of including details. In the event we change the default this is another place to remember to change...

Reply:
For pagination the API documentation is a good source of truth, whereas this is a good place for documenting the difference basic and private information; this part is not covered by the API documentation as it states there is a security scheme based on `Authorization` but not what it's used for.

Reply:
Would be better to have that in the API docs. Not sure if RAML and/or the UI is too limiting.

Reply:
Mostly yes, meant it's not there at the moment, and `profiles` can't know anything about `Authorization` as it's dealt with in the `api-gateway` and hidden from it. If there is any ORCID-specific information however like a mapping between private/trusted/public and what happens with them, it's part of this implementation.

Reply:
I've just updated this. Any better? As it's just documentation, feel free to push your own changes/suggestions.